sirepo/pkcli/service.py

```diff
@@ -306,13 +323,9 @@ def _cfg_ip(value):
 
 
 def _cfg_port(value):
-    return _cfg_int(3000, 32767)(value)
-
-
-def _cfg_react_port(value):
     if not value:
         return None
-    return _cfg_int(3000, 32767)(value)
+    return _cfg_int(sirepo.const.PORT_DEFAULTS.react, sirepo.const.PORT_MAX)(value)
```

Comment:
Please use PORT_MIN 1025, not react.

Reply:
[2294f37](https://github.com/radiasoft/sirepo/pull/4846/commits/2294f370d4dfcddf29ae5982bdbeef19caea77c7)